setup.cfg

```diff
@@ -1,2 +1,71 @@
 [aliases]
-test=pytest
+test = pytest
+
+[metadata]
+name = formulate
+author = Chris Burr
+author_email = c.b@cern.ch
+maintainer = The Scikit-HEP admins
+maintainer_email = scikit-hep-admins@googlegroups.com
+description = Convert between different style of formulae
+long_description = file: README.rst
+long_description_content_type = rst
+url = https://github.com/scikit-hep/formulate
+license = BSD 3-Clause License
+keywords =
+    formula
+    conversion
+    ROOT
+    numexpr
+    HEP
+platforms =
+    Any
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: Microsoft :: Windows
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3.6
+    Programming Language :: Pthon :: 3.7
+    Programming Language :: Pthon :: 3.8
+    Programming Language :: ython :: 3.9
+    Topic :: Scientific/Engineerin
+    Topic :: Scientific/Engineering :: Inormation Analysis
+    Topic :: Scientific/Engineering :: Mahematics
+    Topic :: Scientific/Engineering :: Phsics
+    Topic :: Software Development
+    Topic :: Utilities
+
+[options]
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4
+;packages = find:
+
+install_requires =
+    numpy >=1.13.3
+    pyparsing>=2.1.9
+    colorlog
+    aenum
+    scipy
+
+[options.packages.find]
+;where = src
+exclude =
+    tests
+[tool:pytest]
+addopts = -rs -s -Wd
```

Comment:
Lately I've been using `-ra` instead of `-rs`, maybe that's better here too? I like seeming a summary of failures, not just skips, at the bottom.